classes/sdk-installer.bbclass

```diff
@@ -95,9 +95,12 @@ python do_create_sdk_installer() {
 
 do_image_tar[postfuncs] += "do_create_sdk_installer"
 
-ROOTFS_POSTPROCESS_COMMAND:append:class-sdk = " rename_installer_script"
+ROOTFS_POSTPROCESS_COMMAND:append:class-sdk = " rename_installer_script setup_kernel_source_dir setup_kernel_source_dir"
```

Comment:
@Yoshihiro-Okada-ML Do we need "setup_kernel_source_dir" twice?

Reply:
No, it's simply typo.
Fixed.